### PR TITLE
done with footer changes#1587

### PR DESCRIPTION
**Pull request by Ashmitha2606**
### Pull Request Checklist

- [x] I have added screenshots and videos to show before and after the working of my code.
- [ ] I have ensured that the screen size is set to 100% while making the video.
- [x] I have synced the latest fork with my local repository and resolved any conflicts.
- [ ] I have mentioned the issue number which I created before making this PR .(format to mention issue number is : fixes #(issue number) ) 
- [x] I understand that if any the above conditions are not met , my PR will not be MERGED .

### CODE REVIEW

**Comment by Ashmitha2606** [comment]
@ANSHIKA-26 @hiteashgupta1 do check....

**Comment by Ashmitha2606** [comment]
Since i have completed the tasks @ANSHIKA-26 
could you avoid a level-2 as you said

File: blog.html [hunk 1]
```diff
@@ -1310,38 +1310,39 @@ <h2 class="footer-title secondary-title">Feature Post</h2>
...
-                     </style>
-                 </div>
-
-                 <h2 class="footer-title secondary-title mt-5">Connect with us</h2>
-
-                 <div class="tags social" id="contact">
+              <h2 class="footer-title secondary-title">Popular Tags</h2>
+              <div class="tags">
+                <div class="d-flex-gap flex-wrap">
+                  <a href="#" class="nav-link btn category-btn">Travel</a>
```
Comment (hiteashgupta1) [inline blog.html:1316]: Add blog.html in href on all pages in place of #

**Review by hiteashgupta1 — CHANGES_REQUESTED** [review]

**Comment by Ashmitha2606** [comment]
@hiteashgupta1 done with changes!!

**Pull request merged**